### ISSUE TITLE
Narrator is not reading text on using Caps+right arrow in scan mode 

### ISSUE THREAD

**Issue by xxhice** [OfficeDev/office-ui-fabric-react#9761]
## Codepen demonstrating the issue
 
https://codepen.io/xxhice/pen/NZzERb
 Start Narrator and enable scan mode
 Using Caps+right arrow navigate to Toggle button and text next to it
 Verify whether narrator is reading the text on the first time, it will focus on the label again and read on another Caps+right 

## Actual behavior:
Narrator is not reading text on using Caps+right arrow in scan mode for the first time
## Expected behavior:
Narrator should read all the content that is focused OR focus should not got to it
## User Impact:
Screen reader users face difficulty in using application if narrator do not read the content when they are focused

**Comment by Vitalius1** [comment]
@xxhice I did a little of testing and I can’t seem to be able to repro the same behavior when running locally or on the website’s Toggle demo page. I can repro it only in the codepen you provided. The best I can do is to apply `aria-hidden` on it so that Narrator in scan mode skips it but that needs to be discussed with an accessibility expert.

**Comment by Vitalius1** [comment]
@betrue-final-final, need your opinion in hiding the state text (On/Off) in scan mode of Narrator because it looks like focusing on the pill already reads out the state of the toggle so if you Caps + right through the control you will hear the state twice: once on the pill and once on the state label.

**Comment by Vitalius1** [comment]
I also have been able to repro on https://codesandbox.io/s/kind-colden-f6y4u running the latest OUFR version. But still running it locally or on the website does not repro.

**Comment by xxhice** [comment]
> @xxhice I did a little of testing and I can’t seem to be able to repro the same behavior when running locally or on the website’s Toggle demo page. I can repro it only in the codepen you provided. The best I can do is to apply `aria-hidden` on it so that Narrator in scan mode skips it but that needs to be discussed with an accessibility expert.
> 
> Can you confirm if the issue persists in a different setup rather than a codepen page?

Yes, we got a bug https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/1495139 after accessibility pass test for our new calendar control and we are using this Toggle control from office UI fabric, so I opened this issue with repro in codepen.
The problem is why Narrator will focus on the label twice? The behavior in website seems working fine, but when I 'Export to CodePen', I could repro this issue.

**Comment by Vitalius1** [comment]
I looked into it more and I can not find a solution to why it working on the website and not in the codepen. Which makes me think this could be an issue related to the environment where the control is running or it’s a Narrator bug. I looked at the accessibility tree provided by both the Edge browser and Accessibility Insights tool in both environments and saw no difference. There is just a button followed by  a label element and nothing to cause an extra stop.

I asked a question [here](https://stackoverflow.microsoft.com/questions/157460/narrator-caps-arrow-stops-twice-on-a-label) and hope for some help from the Narrator team.

**Update**: I actually traced the issue all the way to the `Label` component page that has the same behavior in a codepen environment. I also put together a [codepen](https://codepen.io/vitalius1/pen/KjOLqY) with just a native `span` and `label` elements without any Fabric imports and I can see the same issue. So more likely than not it’s just a Narrator bug. Will leave this issue open till I get some response from the Narrator team.
